test_runtime.sh

```diff
@@ -3,11 +3,23 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+BASH="${BASH_VERSION%.*}"
+BASH_MAJOR="${BASH%.*}"
+BASH_MINOR="${BASH#*.}"
+
+if test "${BASH_MAJOR}" -eq 3 && test "${BASH_MINOR}" -eq 0
+then
+	echo "ERROR: ${0} requires Bash version >= 3.1" >&2
+	echo "you're running ${BASH}, which doesn't support += array assignment" >&2
+	exit 1
+fi
+
 RUNTIME="runc"
+TEST_ARGS=('--args' '/runtimetest')
 KEEP=0 # Track whether we keep the test directory around or clean it up
 
 usage() {
-	echo "$0 -r <runtime> -k -h"
+	echo "$0 -d <log-level> -r <runtime> -k -h"
```

Comment:
On Wed, Jun 08, 2016 at 03:42:57PM -0700, Mrunal Patel wrote:
> >  KEEP=0 # Track whether we keep the test directory around or clean it up
> >  
> >  usage() {
> > -	echo "$0 -r <runtime> -k -h"
> > +	echo "$0 -d <log-level> -r <runtime> -k -h"
> 
> WDYT of replacing -d by -l ?

Done in c2beb36 → 2c377ad.